src/shell.js

```diff
@@ -163,6 +176,10 @@ else if (ENVIRONMENT_IS_SHELL) {
 }
 else if (ENVIRONMENT_IS_WEB || ENVIRONMENT_IS_WORKER) {
   Module['read'] = function shell_read(url) {
+    var data = tryParseAsDataURI(url);
```

Comment:
same issue here, I think, with us parsing the uri manually instead of using an xhr?